lib/components/src/tooltip/TooltipLinkList.tsx

```diff
@@ -15,31 +15,49 @@ const List = styled.div<{}>(
   })
 );
 
-export interface Link extends ListItemProps {
+export interface Link extends Omit<ListItemProps, 'onClick'> {
   id: string;
   isGatsby?: boolean;
+  onClick?: (event: SyntheticEvent, item: ListItemProps) => void;
 }
 
 export interface TooltipLinkListProps {
   links: Link[];
   LinkWrapper?: LinkWrapperType;
 }
 
+const Item: FunctionComponent<TooltipLinkListProps['links'][number]> = props => {
+  const { LinkWrapper, onClick: onClickFromProps, ...rest } = props;
+  const { id, title, href, active } = rest;
+  const onClick = useCallback(
+    (event: SyntheticEvent) => {
+      onClickFromProps(event, rest);
```

Comment:
It now gets called with 2 arguments, the first is the event, as it was before, the second (new) argument is the object, with the id, href, etc on it.